src/components/AppView/AppView.tsx

```diff
@@ -102,39 +106,51 @@ class AppView extends React.Component<IAppViewProps, IAppViewState> {
   }
 
   public render() {
-    const { releaseName } = this.props;
-
     if (!this.state.otherResources) {
       return <div>Loading</div>;
     }
+    const { app } = this.props;
+    if (!app) {
+      return <div>Loading</div>;
+    }
     return (
       <section className="AppView padding-b-big">
-        <AppHeader releasename={releaseName} />
         <main>
           <div className="container container-fluid">
```

Comment:
Now, the container is fixed to a certain width. Please, remote the `container-fluid` property